src/AnnotatedExternalLibrary.hs

```diff
@@ -34,6 +34,11 @@ unsafe_unsafe_plus_prop x1 y1 x2 y2 =
 solo_plus :: SDouble Diff s1 -> SDouble Diff s2 -> SDouble Diff (s1 +++ s2)
 solo_plus a b = D_UNSAFE $ unsafe_plus (unSDouble a) (unSDouble b)
 
+-- This is an example of mixed typed function to show how Solo can handle sensitive and non-sensitive types
+-- TODO come up with an example that's less odd
+solo_mixed_types :: SDouble Diff s1 -> SDouble Diff s2 -> Bool -> SDouble Diff (JoinSens s1 s2)
+solo_mixed_types a b chooseA = D_UNSAFE $ max (unSDouble a) (unSDouble b)
+
```

Comment:
TODO add an example where we scale a sdouble by a known number. We will need a KnownNat and use it in the output type to scale.